src/error.js

```diff
@@ -341,6 +341,9 @@ export function getErrorReportUrl(message, filename, line, col, error,
   }
   url += `&jse=${detectedJsEngine}`;
 
+  const experiments = experimentTogglesOrNull();
+  url += `&exps=${JSON.stringify(experiments)}`;
```

Comment:
This would need `encodeURIComponent`. Would a more compact custom encoding make sense?